AlphaWallet/Settings/Types/Constants.swift

```diff
@@ -225,4 +225,11 @@ public struct Constants {
     static let fetchContractDataTimeout = TimeInterval(4)
 
     static let refreshTokensThresholdSec: Int = 1
+
+    enum AmountTextField {
+        static let allowedCharacters: String = {
+            let decimalSeparator = Config.locale.decimalSeparator ?? ""
+            return "0123456789" + decimalSeparator + EtherNumberFormatter.decimalPoint
```

Comment:
Can you check if this might break handling for "," and "."? I forgot if we wanted to let users enter both.

Reply:
that is what was in AmountTextField, just extracted it to constants, no other changes for it.